Makefile

```
@@ -290,7 +300,9 @@ docs_serve:
 
 subsystem-run: test subsystem-clean
 
-test:
+test: rest-api-tests enable-kube-api-for-subsystem kube-api-tests
```

As far as i know makefile don't guaranty the order of the dependencies, if something needs to be done in a specific order then you need to call it with make.
@ronniel1 correct me if i'm wrong.

---

You can force prerequisites order by adding a pipe symbol e.g.
```suggestion
test: rest-api-tests | enable-kube-api-for-subsystem kube-api-tests
```

https://www.gnu.org/software/make/manual/make.pdf